include/srtp_priv.h

```diff
@@ -96,12 +97,6 @@ srtp_err_status_t srtp_steam_init_all_master_keys(
     srtp_master_key_t **keys,
     const unsigned int max_master_keys);
 
-/*
- * srtp_stream_init(s, p) initializes the srtp_stream_t s to
- * use the policy at the location p
- */
-srtp_err_status_t srtp_stream_init(srtp_stream_t srtp, const srtp_policy_t *p);
-
```

Comment:
Original PR added srtp_stream_dealloc() here to make it semi public. Since it is no longer needed I removed and noticed that srtp_stream_init is also semi public but never referenced so thought to remove it from here as well. If you prefer I revert and make a new PR for this then let me know.